opba-protocols/xs2a-protocol/src/main/resources/processes/xs2a/authorization/consent/xs2a-authorize-consent.bpmn20.xml

```diff
@@ -23,7 +23,7 @@
     <exclusiveGateway id="isMultipleScaAvailable" name="Multiple SCA available?" default="oneScaAvailable"/>
     <exclusiveGateway id="isScaFinalized" name="SCA finalized?" default="moreScaChallenges"/>
     <serviceTask id="persistConsentAndContext" name="Persist consent and its context for future service session invocation" flowable:async="true" flowable:exclusive="false" flowable:delegateExpression="${xs2aPersistConsentAndContext}"/>
-    <sequenceFlow id="oneScaAvailable" name="One SCA" sourceRef="isMultipleScaAvailable" targetRef="embeddedScaChallenge"/>
+    <sequenceFlow id="oneScaAvailable" name="One SCA" sourceRef="isMultipleScaAvailable" targetRef="isDecoupledScaSelected"/>
```

Comment:
Please prefer vertical/horizontal lines on the diagram
![Selection_155](https://user-images.githubusercontent.com/18463491/96425452-b6255080-1204-11eb-8632-cb312a9cc7b3.png)

Reply:
vertical/horizontal lines

Reply:
Does not seem so
**Current:**
![Selection_161](https://user-images.githubusercontent.com/18463491/97149676-c0e86400-1775-11eb-8561-e51bcb11f72e.png)

**Expected**
![Selection_160](https://user-images.githubusercontent.com/18463491/97149679-c2199100-1775-11eb-8079-9db614482f1f.png)